src/main/java/io/github/hydos/proton/util/ProtonRegisterUtil.java

```diff
@@ -0,0 +1,86 @@
+package io.github.hydos.proton.util;
+
+import io.github.hydos.proton.Proton;
+import net.minecraft.block.Block;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+public class ProtonRegisterUtil {
+
+    /**
+     * @param name
+     *        Name of item instance to be registered
+     * @param item
+     *        Item instance to be registered
+     * @return Item instanced registered
+     */
+    public static <T extends Item> T registerItems(String name, T item) {
+        return registerItems(Proton.identifier(name), item);
+    }
+
+    /**
+     * @param name
+     *        Identifier of item instance to be registered
+     * @param item
+     *        Item instance to be registered
+     * @return Item instance registered
+     */
+    public static <T extends Item> T registerItems(Identifier name, T item) {
```

Comment:
registerItem**s**? it only registers one

Reply:
~~rename it to `registerItemWithIdentifier`~~